### DISCUSSION TITLE
Cara memperbesar gambar di html format readthedown

### DISCUSSION THREAD

**Discussion by daryraa** [teamalgoritma/community#393] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/78478403/194765364-1096471f-3bfc-4799-9ff0-d230822f0a23.png)
Supaya gambar visualisasinya besar itu bagaimana ya soalnya ini kecil banget

**Comment by jafarhf1** [comment, marked as answer]
Dear @daryraa,
Terkait kendala yang dialami, bisa ditambahkan parameter out.width dibagian chunk optionnya, Misalnya: out.width="100%".

Demikian yang bisa kami sampaikan. Apabila jawaban ini membantu, silahkan mark as answer.

**Reply by daryraa** [reply]
apakah seperti ini?
![image](https://user-images.githubusercontent.com/78478403/194973279-c9c9d3c4-0c68-4a1a-b6a9-b49e2bdd8fcc.png)
jika iya pas di knit gambarnya masih kecil mas

**Reply by jafarhf1** [reply]
Halo @daryraa mungkin bisa dicoba dengan parameter berikut:
` ```{r, out.width = "100%", out.height= "100%", echo = FALSE, fig.align = "center"}`

Semoga membantu.

**Reply by daryraa** [reply]
oke akan saya coba mas terima kasih

**Reply by daryraa** [reply]
halo kak sudha saya coba tapi masih sama 
![image](https://user-images.githubusercontent.com/78478403/194992549-415221fe-e2af-446e-8fed-b55926a9ddb2.png)
![image](https://user-images.githubusercontent.com/78478403/194992582-d6f82801-b8c6-4d67-b7f0-343b957eba08.png)

**Reply by ranydwi** [reply]
Selamat siang Pak Dary, boleh diinfokan/screenshot-kan informasi yaml Bapak di bagian paling atas pada RMarkdown Bapak. Semisal seperti contoh saya di bawah.

Kemudian, saran saya pak, coba bapak tidak perlu gunakan parameter `hjust` dan `layout.exp` nya Pak. Dikarenakan hal tersebut tidak terlalu mandatory. Saya khawatir itu juga berpengaruh. Karena seharusnya secara default, hasil knit kita menampilkan hasil plot dengan ukuran yang cukup besar.

```
---
title: "cek"
date: '2022-10-12'
output: rmdformats::readthedown
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = TRUE)
```
Semoga membantu yaa Pak :)

**Reply by daryraa** [reply]
ini ssanya kak 
![image](https://user-images.githubusercontent.com/78478403/195319692-308683c7-3af8-4795-903b-e02a22f36f22.png)

saya coba tidak menggunakan paramet hjust dan layout exp namun hasilnya sama saja kak
![image](https://user-images.githubusercontent.com/78478403/195319471-d31e6890-5b25-4248-b224-4a6a8c9cdf65.png)

terus bukan gambar itu aja, ada juga yang ini kecil juga kak
![image](https://user-images.githubusercontent.com/78478403/195319588-655b5ab9-96b2-42bb-9621-e1f8df74d032.png)

**Reply by jafarhf1** [reply]
Halo @daryraa,
Berdasarkan pengaturan yaml yang bapak berikan, sepertinya yang menjadi permasalahan adalah karena bapak menggunakan parameter thumbnails dan nilainya true. 
```
output: 
   rmdformats::readthedown:
      .......
      thumbnails: true   
      .......
```
Secara default, parameter tersebut bernilai false sehingga tidak membuat hasil gambar pada knitnya menjadi kecil. Oleh karena itu, saran saya bapak tidak perlu menggunakan parameter tersebut, atau ubah menjadi false sebagaimana nilai defaultnya.

Demikian yang bisa kami sampaikan. Apabila jawaban ini membantu, silahkan mark as answer.

**Reply by daryraa** [reply]
Ohiya kak benar sudah bisa terima kasih ya kak
